server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserMailboxesRoutes.java

```diff
@@ -281,6 +281,7 @@ public void defineMailboxExists() {
                         .statusCode(HttpStatus.NOT_FOUND_404)
                         .type(ErrorType.INVALID_ARGUMENT)
                         .message("Invalid get on user mailboxes")
+                        .cause(new IllegalArgumentException("Mailboxes do not exist"))
```

Comment:
1. It is `Mailbox does not exist`

2. I'm against creating an exception that we did not encountered.

I suggest:

```
throw ErrorResponder.builder()
                        .statusCode(HttpStatus.NOT_FOUND_404)
                        .type(ErrorType.INVALID_ARGUMENT)
                        .message("Mailbox does not exist")
                        .haltError();
```

Reply:
It seem not right: we need provide more `details` error but not skip it or replace it in `message`.
I dont know any better way to do it yet.

Reply:
> we need provide more details

No not really, it is a suggestion from the QA team.

`.message("Invalid get on user mailboxes")` do not means anything to me, that message is useless, we'd rather replace it with our usefull message.

> you might break others test with it.

Which ones?

Reply:
`deleteShouldRemoveMailbox`

Reply:
Probably the error message is not accurate there too